docs/content/how-to/baremetal-none.md

```diff
@@ -0,0 +1,429 @@
+packageserver-6fd9f8479-pbvzl                     0/2     Init:0/1   0               6m22s
+redhat-marketplace-catalog-8cc88f5cb-hbxv9        1/1     Running    0               6m29s
+redhat-operators-catalog-b749d6945-2bx8k          1/1     Running    0               6m29s
+~~~
+
+The hosted cluster's kubeconfig can be extracted as:
+
+~~~sh
+oc extract -n ${CLUSTERS_NAMESPACE} secret/${HOSTED}-admin-kubeconfig --to=- > ${HOSTED}-kubeconfig
+oc get clusterversion --kubeconfig=${HOSTED}-kubeconfig
+~~~
+
+### Adding a bare metal worker
+
+* Download the RHCOS live ISO into the httpd server (for example into `/var/www/html/hypershift-none/` on an apache server hosted at www.example.com)
+
+~~~sh
+mkdir -p /var/www/html/hypershift-none/
+curl -s -o /var/www/html/hypershift-none/rhcos-live.x86_64.iso https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.9/4.9.0/rhcos-live.x86_64.iso
+~~~
+
+* Download the ignition generated in the hostedcluster
+
+~~~sh
+curl -s -k -H "Authorization: Bearer $(oc -n clusters-${HOSTED} get secret $(oc -n clusters-${HOSTED} get secret | grep token-${HOSTED}  | awk '{print $1}') -o jsonpath={.data.token})" https://$(oc get node -o wide | grep master | head -1 | awk '{print $6}'):$(oc -n clusters-${HOSTED} get svc ignition-server -o jsonpath={.spec.ports[0].nodePort})/ignition > /var/www/html/hypershift-none/worker.ign
+~~~
+
+* Modify the RHCOS live ISO to install the worker using that ignition file into the `/dev/sda` device (your milleage may vary)
+
+~~~sh
+podman run --rm -it -v /var/www/html/hypershift-none/:/data:z --workdir /data quay.io/coreos/coreos-installer:release iso customize --live-karg-append="coreos.inst.ignition_url=http://www.example.com/hypershift-none/worker.ign coreos.inst.install_dev=/dev/sda" -o ./rhcos.iso ./rhcos-live.x86_64.iso
+podman run --rm -it -v /var/www/html/hypershift-none/:/data:z --workdir /data quay.io/coreos/coreos-installer:release iso kargs show ./rhcos.iso
+chmod a+r /var/www/html/hypershift-none/rhcos.iso
+~~~
+
+* (Optionally) Check the ISO can be downloaded
+
+~~~sh
+curl -v -o rhcos.iso http://www.example.com/hypershift-none/rhcos.iso
+~~~
+
+* Use the ISO as 'virtual media' using `racadm` (in this case, Dell hardware) and boot the server once using the ISO:
+
+~~~sh
+sudo yum install -y sshpass
+
+export IDRACIP=10.19.136.22
+export SSHPASS="xxx"
+export IDRACUSER="root"
+ipmitool -I lanplus -U ${IDRACUSER} -P ${SSHPASS} -H ${IDRACIP} power off
+racadm="sshpass -e ssh -oStrictHostKeyChecking=no ${IDRACUSER}@${IDRACIP} racadm"
+${racadm} remoteimage -d
+${racadm} remoteimage -c -u "foo" -p "bar" -l http://www.example.com/hypershift-none/rhcos.iso
+
+curl -s -L -k https://github.com/dell/iDRAC-Redfish-Scripting/raw/master/Redfish%20Python/SetNextOneTimeBootVirtualMediaDeviceOemREDFISH.py -o SetNextOneTimeBootVirtualMediaDeviceOemREDFISH.py
+python3 ./SetNextOneTimeBootVirtualMediaDeviceOemREDFISH.py -ip ${IDRACIP} -u ${IDRACUSER} -p ${SSHPASS} -d 1 -r y
```

Comment:
This section could be confusing since it uses a combination of ipmitool, racadm and redfish scripts, is there any way this could be made more simple/consistent - or should we perhaps just say "the iso should be attached to the BMC and booted" since these steps will always depend on the exact hardware?

Reply:
Thanks for the comment Steven. I've modified it, would you mind to take a look again?